ArduPlane/GCS_Mavlink.cpp

```diff
@@ -159,33 +159,25 @@ void GCS_MAVLINK_Plane::send_attitude() const
 
 void GCS_MAVLINK_Plane::send_attitude_target() {
 
-    float target_r = 0.0f;
-    float target_p = 0.0f;
-    float target_y = 0.0f;
-    float throttle = 0.0f;
-    Quaternion quat(0.0f,0.0f,0.0f,0.0f);
+    // Check if the attitude target is valid for reporting
+    if (AP_HAL::millis() - plane.quadplane.last_att_control_ms > 100) return;
```

Comment:
You can stash millis in a const and use it down on 175 to save getting it twice. Return should be on the next line per the style guide.
```suggestion
    const uint32_t now = AP_HAL::millis()
    if (now  - plane.quadplane.last_att_control_ms > 100) {
        return;
    }
```

Reply:
Right, of course, I completely missed the other function call